CMakeLists.txt

```diff
@@ -424,7 +397,8 @@ endif ()
 # TODO: This probably shouldn't be the case and should disable specific
 # callbacks when OpenCV is not available
 if (LBANN_WITH_TBINF AND NOT LBANN_HAS_OPENCV)
-  message(FATAL_ERROR "TensorBoard requires OpenCV. Enable LBANN_WITH_VISION to use TensorBoard.")
+  message(FATAL_ERROR "TensorBoard requires OpenCV. "
```

Comment:
Is this check necessary?  `LBANN_WITH_TBINF` is set to `LBANN_WITH_VISION` (line 135), so this *should* never be triggered

Reply:
That's the default value only. You might manually say `LBANN_WITH_VISION=ON`, `LBANN_WITH_TBINF=OFF` (this should be fine) but you could just as easily say: `LBANN_WITH_VISION=OFF`, `LBANN_WITH_TBINF=ON`. This is the case we're protecting against.